src/Assetic/Filter/JSqueezeFilter.php

```diff
@@ -23,7 +23,20 @@ class JSqueezeFilter implements FilterInterface
 {
     private $singleLine = true;
     private $keepImportantComments = true;
-    private $specialVarRx = \JSqueeze::SPECIAL_VAR_RX;
+    private $classname;
+    private $specialVarRx;
+
+    function __construct() {
+        // JSqueeze is namespaced since 2.x, this works with both 1.x and 2.x
+        // Upstream default for special var is false in 2.x
+        if (class_exists('\\Patchwork\\JSqueeze')) {
+            $this->classname = '\\Patchwork\\JSqueeze';
+            $this->specialVarRx = false;
```

Comment:
I wasn't really sure whether to do that or respect the upstream default for the current jsqueeze. As another idea, maybe make it default false in both cases, as experience seems to have suggested that's the best choice (which is why you changed it upstream in 2.x, right?)